docs/user/configuring-agent.md

```diff
@@ -45,25 +50,25 @@ status:
 
 You can specify extra system infos to be included in the device status by listing them under the `system-info` configuration parameter:
 
-| Extra System Info |
-| ----------------- |
-| `hostname` |
-| `kernel` |
-| `distroName` |
-| `distroVersion` |
-| `productName` |
-| `productSerial` |
-| `productUuid` |
-| `biosVendor` |
-| `biosVersion` |
-| `netInterfaceDefault` |
-| `netIpDefault` |
-| `netMacDefault` |
-| `gpu` |
-| `memoryTotalKb` |
-| `cpuCores` |
-| `cpuProcessors` |
-| `cpuModel` |
+| Extra System Info     | Included by Default |
+|-----------------------|---------------------|
+| `hostname`            | Yes                 |
+| `kernel`              | Yes                 |
+| `distroName`          | Yes                 |
+| `distroVersion`       | Yes                 |
+| `productName`         | Yes                 |
+| `productSerial`       | Yes                 |
+| `productUuid`         | Yes                 |
+| `biosVendor`          | No                  |
+| `biosVersion`         | No                  |
+| `netInterfaceDefault` | Yes                 |
+| `netIpDefault`        | Yes                 |
+| `netMacDefault`       | Yes                 |
+| `gpu`                 | No                  |
+| `memoryTotalKb`       | No                  |
+| `cpuCores`            | No                  |
+| `cpuProcessors`       | No                  |
+| `cpuModel`            | No                  |
```

Comment:
Ah, I understood you wanted to add a description of each key. Maybe these default belong into the re defaults, maybe they belong into the parameter table (line 20)?

Reply:
sounds good will add descriptions here